.github/workflows/smoke-test.yaml

```diff
@@ -0,0 +1,31 @@
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+    branches:
+    - master
+name: Smoke Test
+jobs:
+  smoke-test:
+    runs-on: self-hosted
+    steps:
+      - name: Clean
+        run: rm -rf ${GITHUB_WORKSPACE}/*
+      - name: Checkout build scripts
+        run: git clone ~/harness.git
+      - name: Checkout zbdbench
+        run: git clone ~/zbdbench.git harness/rocksdb-context/zbdbench
+      - name: Checkout rocksdb
```

Comment:
I don't know how the harness works, but can't we have all of the smoke-test specifics in that project? (rocksdb version, how we run the tests)  - now it looks like we have to change both the harness AND this yaml if we want to change how the smoke test is run.

Reply:
Yes, those things live in the test harness. The harness runs the smoke test with default settings. Parameters are not intended to be modified beyond default for this CI check, so there is really nothing to change.

The harness is not public at the moment, but I will look into releasing it, if it makes sense.